### DISCUSSION TITLE
NoBuild

### DISCUSSION THREAD

**Discussion by MyYoungSav** [GriefPrevention/GriefPrevention#1158] [General]
Hi I noticed there is a NoBuild in PvP ingame and in the plugin files, but there is no way to actually access this and turn it off, please fix :D

**Comment by RoboMWM** [comment]
what?

**Reply by MyYoungSav** [reply]
So, there is NoBuild in Pvp right? But no way to turn it off in the config, it just seems to be stuck on.

**Reply by RoboMWM** [reply]
no there isn't.

**Reply by RoboMWM** [reply]
that one discussion result is this one. ![](https://i.imgur.com/lm0UCP8.png)

**Reply by MyYoungSav** [reply]
Ok would it be possible to add that to the plugin? A lot of my players have been asking for it :D

**Comment by bigpresh** [comment]
I *think* the OP is referring [to the check that prevents building while engaged in PvP](https://github.com/TechFortress/GriefPrevention/blob/master/src/main/java/me/ryanhamshire/GriefPrevention/Claim.java#L394-L399) and, indeed, there's no config option to turn off that check.

(EDIT: not sure why GitHub's search feature doesn't find that when you search for "NoBuild"...)

**Reply by RoboMWM** [reply]
github doesn't do partial matching. thanks.
